pkg/webhooks/resource/imageverification/handler.go

```diff
@@ -92,6 +106,8 @@ func (h *imageVerificationHandler) handleVerifyImages(logger logr.Logger, reques
 		}
 	}
 
+	go h.handleAudit(policyContext.NewResource, request, nil, engineResponses...)
```

Comment:
This changes the current processing logic. Defer to @JimBugwadia.

Reply:
It doesn't change the current logic, just adds admission reports in the admission webhook.

Reply:
ok, the name is a bit confusing.

This is invoked in mutating webhook, and the request may be rejected by the validating webhook later. Should we create admission reports in validating webhook instead?

Reply:
As explained in the meeting, we can't because the `imageReference` won't match.

Reply:
But if the request gets blocked, do we still want it be reported?

Reply:
No, reporting is only reserved for extant resources.

Reply:
Moved back to the original line

Reply:
It's just before the return true line, this request is going to be fulfilled.
There is no way to know if the resource will be created in the end as there are other webhooks.